include/server.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef MEOWCHKIN_SERVER_HPP
+#define MEOWCHKIN_SERVER_HPP
+#include <boost/asio.hpp>
+#include <iostream>
+#include <map>
+#include <optional>
+#include <queue>
+#include <string>
+#include <vector>
+
+#include "sended_classes.hpp"
+
+using boost::asio::ip::tcp;
+
+namespace meow {
+class Server {
+  int count_of_handled_clients{0};
```

Comment:
probably we can use `players_info.size()` instead storing additional variable

Reply:
It used to count clients ready for play after handshake